Examples/ObjectiveC/DDSLayerSelectionExample.m

```diff
@@ -0,0 +1,100 @@
+//
+//  DDSLayerSelectionExample.m
+//  Examples
+//
+//  Created by Jordan Kiley on 3/21/17.
+//  Copyright © 2017 Mapbox. All rights reserved.
+//
+
+// url to use: 
+
+#import "DDSLayerSelectionExample.h"
+@import Mapbox;
+
+NSString const *MBXExampleDDSLayerSelection = @"DDSLayerSelectionExample";
+
+@interface DDSLayerSelectionExample () <MGLMapViewDelegate, UIGestureRecognizerDelegate>
+
+@property (nonatomic) MGLMapView *mapView;
+
+@end
+
+@implementation DDSLayerSelectionExample
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    
+    self.mapView = [[MGLMapView alloc] initWithFrame:self.view.bounds];
+    self.mapView.delegate = self;
+    [self.mapView setCenterCoordinate:CLLocationCoordinate2DMake(39.232253141714885,-97.91015624999999)];
```

Comment:
These coordinates are waaaaay too precise — I think this must be sub-nanometer precision. 😁

In general, four or five digits after the decimal place is great. Check out this page for a nice table listing various levels of precision: https://en.wikipedia.org/wiki/Decimal_degrees